examples/gno.land/p/demo/zteams/teams_test.gno

```diff
@@ -0,0 +1,243 @@
+package zteams
+
+import (
+	"std"
+	"gno.land/p/demo/users"
+	"testing"
+)
+
+// Helper function to create a sample user.
+func createUser(address string) users.User {
+	return users.User{
+		Address: std.Address(address),
```

Comment:
When creating addresses in test, it's better to use the `testutils.TestAddress` function.

ref: https://github.com/gnolang/gno/blob/master/docs/concepts/packages.md#testutils

Reply:
done